route53_crossfeed_app.tf

```
@@ -6,32 +6,18 @@
 # Prod entries
 # ------------------------------------------------------------------------------
 
-# The hosted_zone_id for the below records comes from https://docs.aws.amazon.com/general/latest/gr/elb.html
-# (ALBs in us-gov-west-1 region)
-resource "aws_route53_record" "crossfeed_prod_A" {
+# CNAME so crossfeed.cyber.dhs.gov → cyber‑hygiene.cisa.dhs.gov
+resource "aws_route53_record" "crossfeed_prod_CNAME" {
   provider = aws.route53resourcechange
 
-  alias {
-    name                   = "crossfeed-prod-1638162291.us-gov-west-1.elb.amazonaws.com."
-    evaluate_target_health = false
-    zone_id                = "Z33AYJ8TM3BH4J"
-  }
-  name    = "crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
-  type    = "A"
+  name    = "crossfeed.${data.aws_route53_zone.cyber_dhs_gov.name}"
```

I missed this one on my last pass:
```suggestion
  name    = "crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
```